docs/Developer's Guide.md

```diff
@@ -107,12 +114,12 @@ Sometimes, you would like to connect the frontend on your local computer to the
 
    where:
 
-   - `API_VM_PORT_NUMBER` is the port on which the 360 API is running on the VM.
-   - `API_PC_PORT_NUMBER` is the port that you want the API to be sent to on your personal computer (not the VM).
+   - `API_VM_PORT_NUMBER` is the port that you selected for the API on the virtual machine like `http://localhost:[API_VM_PORT_NUMBER]`. Please make sure to use the non-HTTPS version as HTTPS does not work over the SSH tunnel.
+   - `API_PC_PORT_NUMBER` is the port that you want the API to be sent to on your personal computer. Feel free to use the same port as above for this one.
    - `USER` is your account on the host machine.
-   - `IP or HOSTNAME` is the IP or HOSTNAME of your host machine.
+   - `IP or HOSTNAME` is the IP or HOSTNAME of your host machine. You can find this on Windows by typing `ipconfig`, on Linux or Mac with `ifconfig`
 
-You are now able to point your local frontend to the remote backend by setting `VITE_APP_API_URL` to `localhost:[API_PC_PORT_NUMBER]`.
+You are now able to point your local frontend to the remote backend by setting `VITE_APP_API_URL` to `http://localhost:[API_PC_PORT_NUMBER]` in the `.env.development` file. Make sure that you also update http to https if you are running the API over TLS.
```

Comment:
Since TLS doesn't work over SSH, we should probably change this recommendation to agree with the one above about not using HTTPS, right?

Reply:
Yes, thanks for catching that.
```suggestion
You are now able to point your local frontend to the remote backend by setting `VITE_APP_API_URL` to `http://localhost:[API_PC_PORT_NUMBER]` in the `.env.development` file.
```